smart_fetch/tasks/meds.py

```diff
@@ -1,34 +1,12 @@
 from smart_fetch import hydrate_utils, resources
 
 
-class MedsTask(hydrate_utils.Task):
+class MedsTask(hydrate_utils.ReferenceDownloadTask):
```

Comment:
This change and several like it here are unrelated to the main task - I just realized that now that I made a convenience class, I can use it in few more places than I originally did.